vercel_blob/blob_store.py

```
@@ -264,13 +306,14 @@ def delete(url: any, options: dict = {}) -> dict:
             'POST',
             headers=headers,
             json={"urls": [url] if isinstance(url, str) else url},
+            timeout=timeout,
         )
         return _response_handler(resp)
     else:
         raise Exception('url must be a string or a list of strings')
 
 
-def copy(blob_url: str, to_path: str, options: dict = {}) -> dict:
+def copy(blob_url: str, to_path: str, options: dict = {}, timeout: int = 10, verbose: bool = False) -> dict:
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)